op-e2e/bridge_test.go

```diff
@@ -119,3 +125,129 @@ func TestERC20BridgeDeposits(t *testing.T) {
+	require.NoError(t, err)
+	require.NotZero(t, bridgeAmount.Uint64())
+
+	// Send Alice's tokens to Bob
+	require.Nil(t, err)
+	tx, err = transactions.PadGasEstimate(aliceL2Opts, 1.1, func(opts *bind.TransactOpts) (*types.Transaction, error) {
+		return l2Token.Transfer(opts, l2Opts.From, bridgeAmount)
+	})
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l2Client, tx.Hash())
+	require.NoError(t, err)
+
+	bobL1Balance, err := l1Token.BalanceOf(&bind.CallOpts{}, l1Opts.From)
+	require.NoError(t, err)
+	require.Zero(t, bobL1Balance.Uint64())
+	bobL2Balance, err := l2Token.BalanceOf(&bind.CallOpts{}, l2Opts.From)
+	require.NoError(t, err)
+	require.Equal(t, bobL2Balance, bridgeAmount)
+
+	// Withdraw GovernanceToken to L1
+	tx, err = transactions.PadGasEstimate(l2Opts, 1.1, func(opts *bind.TransactOpts) (*types.Transaction, error) {
+		return l2Bridge.BridgeERC20(opts, l2TokenAddr, l1TokenAddr, bobL2Balance, 100000, []byte{})
+	})
+	require.NoError(t, err)
+	receipt, err := wait.ForReceiptOK(context.Background(), l2Client, tx.Hash())
+	require.NoError(t, err)
+	version := eth.OutputVersionV0
+	proveReceipt, finalizeReceipt := ProveAndFinalizeWithdrawal(t, version, cfg, l1Client, sys.EthInstances["verifier"], cfg.Secrets.Bob, receipt)
+	require.Equal(t, types.ReceiptStatusSuccessful, proveReceipt.Status)
+	require.Equal(t, types.ReceiptStatusSuccessful, finalizeReceipt.Status)
+
+	// Withdrawal complete, Should have balance on L1
+	bobL1Balance, err = l1Token.BalanceOf(&bind.CallOpts{}, l1Opts.From)
+	require.NoError(t, err)
+	require.NotZero(t, bobL1Balance.Uint64())
+
+	// Deposit GovernanceToken to L2
+	tx, err = transactions.PadGasEstimate(l1Opts, 1.1, func(opts *bind.TransactOpts) (*types.Transaction, error) {
+		return l1Bridge.BridgeERC20(opts, l1TokenAddr, l2TokenAddr, bobL1Balance, 100000, []byte{})
+	})
+	require.NoError(t, err)
+	depositReceipt, err := wait.ForReceiptOK(context.Background(), l1Client, tx.Hash())
+	require.NoError(t, err)
+
+	t.Log("Deposit through L1StandardBridge", "gas used", depositReceipt.GasUsed)
+
+	// compute the deposit transaction hash + poll for it
+	portal, err := bindings.NewKromaPortal(cfg.L1Deployments.KromaPortalProxy, l1Client)
+	require.NoError(t, err)
+
+	depIt, err := portal.FilterTransactionDeposited(&bind.FilterOpts{Start: 0}, nil, nil, nil)
+	require.NoError(t, err)
+	var depositEvent *bindings.KromaPortalTransactionDeposited
+	for depIt.Next() {
+		depositEvent = depIt.Event
+	}
+	require.NotNil(t, depositEvent)
+
+	depositTx, err := derive.UnmarshalDepositLogEvent(&depositEvent.Raw)
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l2Client, types.NewTx(depositTx).Hash())
+	require.NoError(t, err)
+
+	// Should have balance on L2
+	bobL2Balance, err = l2Token.BalanceOf(&bind.CallOpts{}, l2Opts.From)
+	require.NoError(t, err)
+	require.Equal(t, bobL1Balance, bobL2Balance)
```

Comment:
And please check the balance of Bob on L1 is zero.
Further more, let's check the total supply on L1 is decreased by the amount of deposit.